test/appium/views/home_view.py

```diff
@@ -205,11 +205,10 @@ def open_status_test_dapp(self, allow_all=True):
         dapp_view = self.dapp_tab_button.click()
         dapp_view.open_url(test_dapp_url)
         status_test_dapp = dapp_view.get_status_test_dapp_view()
-        for _ in range(2):
-            if allow_all:
-                status_test_dapp.allow_button.click()
-            else:
-                status_test_dapp.deny_button.click()
+        if allow_all:
```

Comment:
Why removed 'Allow'/'Deny' click twice here? In simpledapp we allow access to wallet and then with the second window, access to public key

Reply:
Ah I see click_until_absense_of_element does the thing, ok.